event/src/main/java/com/bazaarvoice/emodb/event/db/astyanax/Constants.java

```diff
@@ -20,7 +20,7 @@
     static final Duration OPEN_SLAB_MARKER_TTL = Duration.ofMinutes(20);
 
     /** Don't keep slabs open for too long.  Rotate them periodically. */
-    static final Duration SLAB_ROTATE_TTL = Duration.ofDays(1);
+    static final Duration SLAB_ROTATE_TTL = Duration.ofHours(1);
```

Comment:
minor: do you want to add a comment? about changing from previous value of 1 day to 1 hour and why.